static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -45,6 +45,27 @@ function GroupEventEntry({event, entryType, group, project}: GroupEventEntryProp
   const organization = useOrganization();
   const matchingEntry = event.entries.find(entry => entry.type === entryType);
 
+  if (entryType === EntryType.REPLAY) {
+    const replayId = event?.tags?.find(({key}) => key === 'replayId')?.value;
+    if (!replayId) {
+      return null;
+    }
+
+    return (
+      <EventEntry
+        projectSlug={project.slug}
+        group={group}
+        entry={{
+          data: {
+            replayId,
+          },
+          type: EntryType.REPLAY,
+        }}
+        {...{organization, event}}
+      />
+    );
+  }
+
```

Comment:
This is a temporary shim until the issues have a not-empty `entries:[]` array. See the `EntryReplay` shape above too

Reply:
I will ask about this when the issues-platform team is online but my thoughts this morning is that "entries" is not the correct place for this data.  `evidenceData` seems like a much better place.  But we will see what the issues team says.